mne/io/eyelink/eyelink.py

```diff
@@ -398,7 +402,7 @@ def _get_recording_datetime(self):
                         # Even though dt is probably in local time zone.
                         dt_naive = datetime.strptime(dt_str, fmt)
                         dt_aware = dt_naive.replace(tzinfo=tz)
-                        self._meas_date = dt_aware
+                        self._raw_extras[0]["dt"] = dt_aware
                         break
```

Comment:
And ideally this would become a private function like `def _get_recording_datetime(fname): ... retun dt` rather than a method